### ISSUE TITLE
Using Batch Norm after each Convolution layer

### ISSUE THREAD

**Issue by Kannan-Balakrishnan** [deepsphere/deepsphere-pytorch#12]
Good Evening,
I appreciate your great work on Spherical graphs. I am using the Chebchev Convolution (from pytorch) with symmetric normalization. However, in our case, we also use batch norm or instance norm after each convolution layers. My question is, does it make sense to have a Chebchev convolution with symmetric normalization followed by a batch norm or an instance norm? Does it help in faster convergence?

Thanks for you time :)
